tests/testStreamClosedMethods.cpp

```diff
@@ -301,7 +299,7 @@ TEST_F(StreamClosedReturnValues, WaitForStateChangeReturnsClosed){
     ASSERT_TRUE(openAndCloseStream());
     StreamState next;
     Result r = mStream->waitForStateChange(StreamState::Open, &next, 0);
-    ASSERT_EQ(r, Result::ErrorClosed) << convertToText(r);
+    EXPECT_TRUE(r == Result::OK || r == Result::ErrorClosed) << convertToText(r);
```

Comment:
The idiom here is r == Result::OK || r == Result::ErrorClosed. This is used throughout our tests